### ISSUE TITLE
Feature idea: Implement WideNES for widescreen gameplay?

### ISSUE THREAD

**Issue by greigs** [bfirsh/jsnes#300]
based on the work done by @daniel5151 http://prilik.com/blog/2018/08/24/wideNES.html
I am considering attempting this on my [new fork](https://github.com/greigs/jsnes), but would like to gauge interest first. Thumbs up and follow my fork if this of interest to you.

**Comment by daniel5151** [comment]
I'd also recommend taking a gander at WideGB (https://kemenaran.winosx.com/posts/widegb-playing-game-boy-games-on-wide-screens/), a similar project for the original Gameboy [Color]. That said, do note that the Gameboy lends itself better to this sort of technique, since games aren't as likely to rely on mid-frame IRQs to create status bars / cool effect sand such.

Best of luck! I'd love to see something playable in the browser :smile:

**Comment by greigs** [comment]
Yeah NES definitely seems trickier than GB. I suspect based on your findings there would need to be some game-by-game database of settings. Maybe the wide feature will be an opt-in option only on games that are tested to support it.
